### PR TITLE
Title definitions

### PR DESCRIPTION
**Pull request by haydarkalca**

### CODE REVIEW

**Comment by hakankecelioglu** [comment]
@haydarkalca plese resolve conflict

**Comment by oguzhanyalcin** [comment]
@hakankecelioglu its reviewers job to merge the branch. Please merge it...

File: migrations/mysql/sqls/20160220230514-generate-title-table-up.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE DEF_TITLE (
```
Comment (aliemrahpekesen) [inline migrations/mysql/sqls/20160220230514-generate-title-table-up.sql]: Please add if exists check for creating table.

Comment (haydarkalca) [inline migrations/mysql/sqls/20160220230514-generate-title-table-up.sql]: added if not exists check

**Pull request merged**